workers/vita/Dockerfile

```diff
@@ -17,56 +17,77 @@ RUN sed 's/^deb \(.*\)/deb-src \1/' /etc/apt/sources.list \
 ENV VITASDK=/opt/vitasdk
 ENV PATH=$PATH:$VITASDK/bin
 
-ARG URL=https://github.com/vitasdk/autobuilds/releases/download/master-linux-v567/vitasdk-x86_64-linux-gnu-2017-10-29_09-48-44.tar.bz2
+#Install VitaSDK base
+ARG URL=https://github.com/vitasdk/autobuilds/releases/download/master-linux-v611/vitasdk-x86_64-linux-gnu-2017-12-01_14-01-36.tar.bz2
 
 RUN wget --progress=bar:force:noscroll -O - $URL \
 	| tar --no-same-owner --no-same-permissions -C /opt/ -xjf -
 
-# These insecure, unversioned downloads are not great, but OpenSSL and SDL2 are
-# not trivially buildable for this target so we use these packages for the
-# moment. At the time of the first build these corresponded to
-# https://github.com/vitasdk/packages/commit/b62cb68c349f8f389a4b4e57356c398c43f5531d
-# in case someone needs to rebuild them from source later
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/openssl.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/curl.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/sdl2.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/sdl_net.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-
+#Install git
+RUN apt-get -y install git
+
+#Install sudo
+RUN apt-get -y install sudo
+
+#Install curl
+RUN apt-get -y install curl
+
+#Install Vita versions of most common libraries, including ogg, png, zlib etc.
+RUN git clone https://github.com/vitasdk/vdpm && \
```

Comment:
Bulk installation of libraries is not an appropriate way to install software for a Docker image. These images should contain only what’s needed to build ScummVM.

Furthermore, this command:

1. Obfuscates the fact that these packages are insecurely downloaded and are vulnerable to MITM;
1. Downloads an entire Git repository of data when only one commit is needed (GitHub has the ability to send any single commit by URL which is used already in some of the other Dockerfiles);
1. Does not pin to a specific commit to ensure [build reproducibility](https://en.wikipedia.org/wiki/Deterministic_compilation).

So, it is not appropriate to use this.